docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -67,221 +103,338 @@ hypershift create cluster kubevirt \
+And the nodepool will be in the desired state:
 
-Every OpenShift cluster comes setup with a default application ingress
-controller which is expected have an wildcard DNS record associated with it.
-By default, guest clusters created using the Hypershift KubeVirt provider
-will automatically become a subdomain of the underlying OCP cluster that
-the KubeVirt VMs run on.
+```shell
+oc get nodepools --namespace clusters
 
-For example, if an OCP cluster cluster has a default ingress DNS entry of
-`*.apps.mgmt-cluster.example.com`, then the default ingress of a KubeVirt
-guest cluster named `guest` running on that underlying OCP cluster will
-be `*.apps.guest.apps.mgmt-cluster.example.com`.
+NAME                      CLUSTER         DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION   UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+example                   example         5               5               False         False        4.12.7                                       
+example-extra-cpu         example         2               2               False         False        4.12.7  
+```
 
-!!! note
+## Delete a HostedCluster
 
-    For this default ingress DNS to work properly, the underlying cluster
-    hosting the KubeVirt VMs must allow wildcard DNS routes. This can be
-    configured using the following cli command. ```oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'```
+To delete a HostedCluster:
 
-## Customized Ingress and DNS Behavior
+```shell
+hypershift destroy cluster kubevirt --name $HOSTEDCLUSTER_NAME
+```
 
-In lieu of the default ingress and DNS behavior, it is also possible to
-configure a Hypershift KubeVirt guest cluster with a unique base domain
-at creation time. This option does require some manual configuration
-steps during creation though.
+## Advanced Ingress and DNS
 
-This process involves three steps.
+Every OpenShift cluster comes setup with a default application ingress
+controller which is expected have an external DNS record associated with it.
 
-**Step 1. Cluster Creation**
+For example, a HyperShift cluster named `example` with the base domain
+`hypershift.lab` is created, then it is expected that the wildcard domain
+`*.apps.example.hypershift.lab` can be resolved.
 
-Create Hypershift KubeVirt cluster with a custom base domain you control. This
-can be achieved using the `--base-domain` cli argument during cluster creation.
+There are two ways to set this up:
 
-Below is an example.
+* Default: Reuse the management cluster's wildcard DNS routing and make the
+Hypershift KubeVirt cluster a subdomain of the management cluster. This is what we have seen in the previous section.
+* Advanced: Set up a new LoadBalancer service and wildcard DNS record for the HostedCluster `*.apps` domain.
+
+### Deploying the HostedCluster specifying our base domain
```

Comment:
Previously we had a `Default Ingress and DNS Behavior` section which clearly outlined how ingress works out of the box. then we have a `Customized Ingress and DNS Behavior` which went into the details around how someone can set up their own basedomain. I'd like to keep those two sections.

Can the `Default Ingress and DNS Behavior` be re-introduced?

Reply:
Reintroduced and reworded some sections to match with the 3-steps proposed by the original doc:

1. Cluster creation
2. LoadBalancer creation
3. Wildcard DNS configuration